app/cells/placeholder_users/row_cell.rb

```diff
@@ -46,11 +46,20 @@ def button_links
     end
 
     def delete_link
-      return nil unless PlaceholderUsers::DeleteContract.deletion_allowed?(User.current)
+      if PlaceholderUsers::DeleteContract.deletion_allowed?(placeholder_user,
+                                                            User.current,
+                                                            table.user_allowed_service)
 
-      link_to '',
-              deletion_info_placeholder_user_path(placeholder_user),
-              class: 'icon icon-delete'
+        link_to deletion_info_placeholder_user_path(placeholder_user) do
+          "<span class=\"tooltip--left\" data-tooltip=\"#{I18n.t('placeholder_users.delete_tooltip')}\"><i class=\"icon icon-delete\"></i></span>".html_safe
```

Comment:
I would not have put the tooltip for the delete icon. However, I wanted some kind of hint the other case, the one when you do not have the permission to delete (question mark icon). Then I wanted to provide some hint to the user why deleting that placeholder isn't possible. However, for some weird CSS reason using the tooltip stuff adds some padding to the right, which I didn't know how to fix. The delete icon and the question mark icon didn't align anymore. So the most straight forward solution was to add a tooltip for the delete button, too.